_scripts/baseurl.sed

```diff
@@ -0,0 +1,5 @@
+# Sets the baseurl to an empty string to be used before deploying to s3
+# command to implement only for the _config.yml file is:
+# sed -f _scripts/baseurl.sed _config.yml > _scripts/_config.yml
+
+s/\/m-lab\.github\.io//g;
```

Comment:
Can we do this inline instead of having a separate script file? Also, would prefer if we used a regex to match the entire line (including baseurl) to reduce the chances of this step clobbering `m-lab.github.io` that might appear on another line.